urdf_root: /home/michel/Developer/urdf_zoo
sim_gui: True
rtf: -1.0
